nntrainer/src/neuralnet.cpp

```diff
@@ -91,19 +91,7 @@ std::vector<std::string> parseLayerName(std::string ll) {
   return ret;
 }
 
-NeuralNetwork::NeuralNetwork() {
-  batch_size = 0;
-  learning_rate = 0.0;
-  decay_rate = 0.0;
-  decay_steps = 0.0;
-  epoch = 0;
-  loss = 0.0;
-  cost = COST_UNKNOWN;
-  weight_ini = WEIGHT_UNKNOWN;
-  net_type = NET_UNKNOWN;
-  data_buffer = NULL;
-  config = "";
-}
+NeuralNetwork::NeuralNetwork() : NeuralNetwork("") {}
```

Comment:
These variables should be initialized. Otherwise,  "Analysishub service" will give us error.